go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/containers/image/v5 v5.5.2
 	github.com/containers/ocicrypt v1.0.3
 	github.com/containers/storage v1.23.5
+	github.com/cyphar/filepath-securejoin v0.2.2
```

Comment:
@nalind We had securejoin at one point and then removed it.  Any objections to it going back in?  I think we removed it as we didn't really need it in the places we had it prior, but could be mis-remembering.

Reply:
Well, the docs for it say it's vulnerable to toc-tou problems when another process has write access to the directory tree at the same time that securejoin's poking at it, which is a thing we have to worry about.  If we add it, it's something we'll have to replace before too long.  I don't have an alternative ready to go, though.

Reply:
Didn't something get added to the kernel to eliminate this?

Reply:
`openat2(2)` with the `RESOLVE_IN_ROOT` flag?